src/main/java/subway/path/application/PathService.java

```diff
@@ -40,24 +45,28 @@ public PathResponse searchPathAndFare(LoginMember loginMember, Long source, Long
                 .collect(Collectors.toList()));
     }
 
-    private PathDto searchShortestPath(Long source, Long target) {
+    protected PathDto searchShortestPath(Long source, Long target) {
         DijkstraShortestPath<Long, Integer> dijkstraShortestPath = new DijkstraShortestPath(createGraph());
```

Comment:
지난 PR에서 빠뜨린 내용인데, (_ _) 이 부분에서 `Unchecked assignment` 경고가 발생해요. 
`createGraph`의 결과인 `WeightedMultigraph<Long, DefaultWeightedEdge>`를  생성자 `DijkstraShortestPath(Graph<V, E> graph)`에 담으려면
V=`Long`, E=`DefaultWeightedEdge`가 되어야 하는데,
`DijkstraShortestPath<Long, Integer> dijkstraShortestPath`에서 
V=`Long`, E=`Integer`라서 Parameterized Generic(< > 가 있는 형태의 객체)을 Raw type(< > 가 없는 형태)으로 assign하고 있기 때문이예요

이걸 단순히,
```suggestion
        DijkstraShortestPath<Long, Integer> dijkstraShortestPath = new DijkstraShortestPath<>(createGraph());
```
이렇게 변경하면 이번에는 `Cannot infer arguments` 오류가 발생하겠죠.
저 다익스트라 최단 경로 객체는 V=`Long`, E=`DefaultWeightedEdge`가 맞아요. 😄 
(`Integer`는 어디서 나온지 모르겠지만)

그래서 아래와 같이 되어야 합니다.
```suggestion
        DijkstraShortestPath<Long, DefaultWeightedEdge> dijkstraShortestPath = new DijkstraShortestPath<>(createGraph());
```

Reply:
앗 왜 Integer로 썼는지 모르겠네요😭 인텔리제이에서 경고 표시해주는 것을 눈여겨 봐야 할 것 같아요. 감사합니다!!